lib/faye/redis.rb

```diff
@@ -180,15 +180,20 @@ def redis
         @subscriber.on(:disconnected) do
           @server.info "Faye::Redis: redis pubsub connection disconnected"
         end
-       @subscriber.on(:reconnected) do
+        @subscriber.on(:reconnected) do
           @server.info "Faye::Redis: redis pubsub connection reconnected"
         end
         @subscriber.on(:reconnect_failed) do |count|
-          @server.info "Faye::Redis: redis pubsub connection reconnect failed (#{count}/4)"
+          @server.info "Faye::Redis: redis pubsub connection reconnect failed #{count} time(s)"
```

Comment:
I removed the "/4" part because even after 4 errors even if the connection is set as failed which means queued commands are emptied and subsequent commands not queued and failed, em-hiredis client still tries to reconnect infinitely every 0.5s